src/content/structured/components/toggle-buttons/code.mdx

```diff
@@ -233,43 +237,161 @@ export const snippetsLight = [
 export const snippetsFullWidth = [
   {
     language: "Web component",
-    snippet: `<ic-toggle-button label="Switch to oat milk" full-width="true"></ic-toggle-button>`,
+    snippet: `<ic-toggle-button-group label="Milk selection" name="full width group" full-width="true">
+  <ic-toggle-button label="Switch to oat milk" ></ic-toggle-button>
+  <ic-toggle-button label="Switch to almond milk" ></ic-toggle-button>
+  <ic-toggle-button label="Switch to coconut milk" ></ic-toggle-button>
+  <ic-toggle-button label="Switch to rice milk" ></ic-toggle-button>
+</ic-toggle-button-group>`,
   },
   {
     language: "React",
-    snippet: `<IcToggleButton label="Switch to oat milk" fullWidth />`,
+    snippet: `<IcToggleButtonGroup fullWidth>
```

Comment:
I think making this a selectType="single" would make more sense as a use case fir selecting a single milk choice.